### PR TITLE
all: add account ID to all nodes of a scan

### PR DESCRIPTION
**Pull request by manelmontilla**
This PR modifies the prostprocess logic so it adds the account id to all the nodes produced by an Altimeter scan.
Note that this PR depends on the PR:
https://github.com/adevinta/graph-altimeter/pull/19

### CODE REVIEW

File: graph_altimeter/scan/postprocess.py [hunk 1]
```diff
@@ -6,12 +6,26 @@
...
+    # big number of nodes, we should improve this code even if we can only
+    # reduce the constants of the linear part of the complexity.
     expand_iam_policies(graph_dict)
     _fix_orphan_edges(graph_dict, scan_id)
+    _add_account_id(graph_dict, account_id)
+
+
+def _add_account_id(graph_dict, account_id):
+    """Adds the property account_id to all the vertices in the graph
+    dictionary"""
```
Comment (jroimartin) [inline graph_altimeter/scan/postprocess.py]: Missing trailing dot.

**Review by jroimartin — CHANGES_REQUESTED** [review]
LGTM. Just a tiny typo.

File: graph_altimeter/scan/postprocess.py (hunk 1 above)
Comment (manelmontilla) [reply graph_altimeter/scan/postprocess.py]: True thank you.

**Review by jroimartin — APPROVED** [review]
LGTM

**Pull request merged**